docs/user_guide_splinter.rst

```diff
@@ -0,0 +1,290 @@
+User Guide (Splinter)
```

Comment:
There's a lot of duplicated content in this page. I wonder if it makes sense to make the user guide more generic, and have another place in the documentation that explains how to instantiate/use the different drivers.